backend/devices/bitbox02/protobuf.go

```
@@ -51,6 +51,9 @@ var ethMsgCoinMap = map[coin.Code]messages.ETHCoin{
 	"eth-erc20-bat":       messages.ETHCoin_ETH,
 	"eth-erc20-mkr":       messages.ETHCoin_ETH,
 	"eth-erc20-zrx":       messages.ETHCoin_ETH,
+	"eth-erc20-wbtc":      messages.ETHCoin_ETH,
+	"eth-erc20-paxg":      messages.ETHCoin_ETH,
+	"eth-erc20-dgx":       messages.ETHCoin_ETH,
```

there was a squash error here i think, this wasn't supposed to be here